modules/sync/unique_queue.go

```diff
@@ -1,10 +1,13 @@
 // Copyright 2016 The Gogs Authors. All rights reserved.
+// Copyright 2019 The Gogs Authors. All rights reserved.
```

Comment:
Still going through this, but a tiny nit so far is the copyright.

Reply:
```suggestion
// Copyright 2019 The Gitea Authors. All rights reserved.
```